packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/collection..st

```diff
@@ -0,0 +1,4 @@
+accessing
+collection: aCollection
```

Comment:
Maybe we could name the accessors `text:` and `text ` to specify that the variable contains the text of the document (still using the instance var `collection`).